interative/interative.go

```diff
@@ -9,6 +9,7 @@ import (
 	"strings"
 
 	"github.com/chzyer/readline"
+	"github.com/rumlang/rum/libs"
```

Comment:
lib is better for package name

Reply:
example: https://github.com/rumlang/encoding